packages/openactive-integration-tests/test/features/payment/payment-reconciliation-detail-validation/not-implemented/payment-reconciliation-detail-validation-missing-accountId-test.js

```diff
@@ -0,0 +1,14 @@
+const { FeatureHelper } = require('../../../../helpers/feature-helper');
+const { notImplementedTest } = require('../common');
+
+FeatureHelper.describeFeature(module, {
+  testCategory: 'payment',
+  testFeature: 'payment-reconciliation-detail-validation',
+  testFeatureImplemented: false,
+  testIdentifier: 'payment-reconciliation-detail-validation-missing-accountId',
```

Comment:
```suggestion
  testIdentifier: 'payment-reconciliation-detail-validation-missing-reconciliation-details',
```

And rename file to match

Reply:
Done